source/mapping-rules/new-rules-suggested-mappings.txt

```diff
@@ -56,6 +56,12 @@ Steps
       These mapping rules are not added to your project until you click the 
       :guilabel:`Apply` button.
 
+      - If the collection entity doesn't exist, it is added to the 
```

Comment:
I think we only allow suggesting from existing collections. What is the action being desscribed here?

Reply:
On reading this again, I think it makes more sense to remove both bullet points. I think the original note directive covers this effectively.